Source/RxCocoa/UIScrollView+reachedBottom.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UIScrollView+reachedBottom.swift
+//  RxSwiftExt
+//
+//  Created by Anton Nazarov on 09/05/2019.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+public extension Reactive where Base: UIScrollView {
+    /**
+     Shows if the bottom of the UIScrollView is reached.
+     - parameter offset: A threshhold indicating the bottom of the UIScrollView.
+     - returns: ControlEvent that emits when the bottom of the base UIScrollView is reached.
+     */
+    func reachedBottom(offset: CGFloat = 0.0) -> ControlEvent<Void> {
+        let source = contentOffset.map { [base] contentOffset in
```

Comment:
I'm pretty sure capturing `base` this way is basically like not doing anything :) 
Either remove it completely like you wanted or make it a `weak base`.